snapshotTestContent/SharedMatrixDeletes/current_snapshots/snapshotVersion.json

```diff
@@ -1 +1 @@
-{"snapshotVersion":"2.0.0-internal.1.0.0"}
\ No newline at end of file
+{"snapshotVersion":"2.0.0-internal.3.1.0"}
```

Comment:
The current_snapshots for "SharedMatrixDeletes" was not updated. It's because of this bug - https://dev.azure.com/fluidframework/internal/_workitems/edit/233. You will have to hack around a little to update these.